### DISCUSSION TITLE
I am getting this error

### DISCUSSION THREAD

**Discussion by ktroberson** [Security-Onion-Solutions/securityonion#1615] [Unsupported Versions]
rendering SLS 'base:thehive' failed: Jinja variable 'list object' has no attribute 'values'

I built a master node in VM workstation then exported to airgapped environment and now I am getting this error. Hope you guys have answer for me.

**Comment by TOoSmOotH** [comment]
Is the IP the same?

**Reply by ktroberson** [reply]
Yes

**Reply by ktroberson** [reply]
The only thing that has changed is the gateway and the dns server

**Comment by m0duspwnens** [comment]
@ktroberson 
Do you get this error every time you run a `salt-call state.highstate` or is it intermittent?

What if you just run a `salt-call state.apply thehive`?

**Reply by ktroberson** [reply]
state.highstate not available  is the response

**Comment by ktroberson** [comment]
On startup I get errors saying that it cannot find the dockers containers for elastic, kibana, logstash and a couple others. But I can see the containers in my vm. So I do so-elasticsearch-restart and I get this error

Keith Roberson
678-410-8830

**Reply by ktroberson** [reply]
This error is just not for the hive, its for all the containers. I built the Management node in VM workstation 15 under esxi 6.5, transferred the vm to my airgap environment. Upon startup I run so-status and I have failures and missing status in all the containers. I can see the containers in the vm. For some reason its not pulling them. When I built the VM in Workstation I also enabled so-elastic-full. I verified that it ran before cloning it and moving it to disk. I am using the same IP address. But I believe it has something to do with pillar and the salt stack.

**Comment by jertel** [comment]
I noticed the Jinja error today on one of my VMs. The salt-minion process was unresponsive. Killing that process/PID and re-starting it allowed my Docker containers to begin coming online. When I restarted salt-minion I ran it in the terminal with debug enabled, just in case there was some console output that would shed some light on the issue, but once restarted it behaved normally so I couldn't reproduce the unresponsive state.

If you're still seeing this I'd be interested to know if restarting salt-minion helps in your situation as well.
